.bowerrc

```diff
@@ -1,3 +1,3 @@
 {
-  "directory" : "/_sass"
-}
\ No newline at end of file
+  "directory" : "public/components"
+}
```

Comment:
@scmx: @MrMagnus had some issues with permissions so I told him to add this if there wasn't a file already that defined a directory.

However. As you say this change will break stuff. But the previous one isn't correct. when the value is `/_sass` it wants to create that folder in my computers root directory 😄. I'll will reverse this change and fix the old one.